README.md

```diff
@@ -1,33 +1,63 @@
-# Using this template
+# seidemann-lab-to-nwb
+NWB conversion scripts for Seidemann lab data to the [neuro data without borders](https://www.nwb.org/) data format.
 
-This repo serves as a generic template for all conversion projects undertaken by the CatalystNeuro team. You can easily use this by...
+## Clone and dev install
+To run the following repo you need to have installed both git and pip then do the following.
 
-Creating a new project under the CatalystNeuro umbrella (or fork this repo and use it for your personal repos)...
-
-![image](https://user-images.githubusercontent.com/51133164/167436507-8b30ca9d-588c-4941-8385-bdff050f6558.png)
-
-
-Select this template at the top...
-
-![image](https://user-images.githubusercontent.com/51133164/167436692-670480f1-216d-4cac-a1b3-76ad518ec2f6.png)
-
-
-
-# Setting up pre-commit bot
-
-For new repos, an extra step of allowing the black auto-commit CI bot may have to be enabled at: https://results.pre-commit.ci/
-
-
-
-# Dataset-specific environments
-
-If the conversion involves multiple very different datasets collected over several years it is recommended to have separate environments for each dataset to reconcile dependency conflicts, instead of attempting to either keep old conversions up to date with recent upstream changes or using strong version pinning at the outer level `my-lab-to-nwb` requirements.
+```
+$ git clone https://github.com/catalystneuro/seidemann-lab-to-nwb
+$ pip install -e .
+```
 
-These environments can be easily made by simply calling
+Alternatively, to clone the repository and set up a conda environment, do:
+```
+$ git clone https://github.com/catalystneuro/seidemann-lab-to-nwb
+$ cd nwb-conversion-tools
```

Comment:
How are they `cd`'ing into `nwb-conversion-tools`? Are they recommended to use the latest clone of that as well? (might have to until we get a new release cut, but ideally they'd be able to use the PyPI version)

Reply:
This is a mistake. It should be the name of the repo. Will be propagated to cookiecutter.